jackson-simple/src/test/java/com/baeldung/jackson/date/Event.java

```diff
@@ -4,17 +4,17 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 
-public class EventWithFormat {
+public class Event {
     public String name;
 
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
     public Date eventDate;
 
-    public EventWithFormat() {
+    public Event() {
```

Comment:
please check the updated article: https://www.baeldung.com/jackson-annotations 
the Event classes should not be changed here

Reply:
Done, reverted the rename